### PR TITLE
Qr code scanner

### PR DESCRIPTION
**Pull request by cyBerta**

### CODE REVIEW

**Comment by r10s** [comment]
- in the simulator, leaving the qr-code-scan-viewcntroller takes some time. it's fine on a native device, however, this might be worth a look. maybe sth. is wrong.

- i think the qr-code-scan-viewcntroller should have a title and the same hint as on android (currently just "place you camera ober the QR code", but we will probably refine this at some point)

File: deltachat-ios/Helper/UIView+Extension.swift [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+//  deltachat-ios
+//
+//  Created by Macci on 05.08.19.
+//  Copyright © 2019 Jonas Reinsch. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+	func makeBorder(color: UIColor = UIColor.red) {
```
Comment (r10s) [inline deltachat-ios/Helper/UIView+Extension.swift]: also here, please indent by 4 spaces

File: deltachat-ios/Helper/UIView+Extension.swift [hunk 2]
```diff
@@ -0,0 +1,96 @@
+//
```
Comment (r10s) [inline deltachat-ios/Helper/UIView+Extension.swift]: please remove the boilerplate

File: deltachat-ios/Helper/UIView+Extension.swift [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+//  File.swift
```
Comment (r10s) [inline deltachat-ios/Helper/UIView+Extension.swift]: no boilerplate, please

File: deltachat-ios/Helper/UIView+Extension.swift (hunk 1 above)
Comment (cyBerta) [reply deltachat-ios/Helper/UIView+Extension.swift]: fixed

File: deltachat-ios/Helper/UIView+Extension.swift (hunk 3 above)
Comment (cyBerta) [reply deltachat-ios/Helper/UIView+Extension.swift]: fixed

**Comment by cyBerta** [comment]
>     * i think the qr-code-scan-viewcntroller should have a title and the same hint as on android (currently just "place you camera ober the QR code", but we will probably refine this at some point)

done

**Comment by cyBerta** [comment]
For now only contact verification is implemented, not group invites

**Comment by r10s** [comment]
got a black screen after a qr code is recognized. however, merging as this pr also provides some other tools and cleanup.

**Comment by r10s** [comment]
> got a black screen after a qr code is recognized. however, merging as this pr also provides some other tools and cleanup.

@cyBerta not sure what was going on there, maybe i had not checked out the latest commits. works like a charme, very nice :)

**Pull request merged**